src/Data/filterDataItems.ts

```diff
@@ -0,0 +1,61 @@
+import { DataConnectionFilters, DataConnectionResultItem } from 'scrivito'
+
+export function filterDataItems(
```

Comment:
* [ ] FYI The location and name of the file looks innocent but may lead to code with bad performance. A reader may not be aware that client-side filtering is a bad idea in most cases and only makes sense for external data sets with a constant fetch effort. Maybe we can find a name that indicates the intended use better? Something like `filterDataItemsAfterFullFetch`? Maybe we could even ask the respective backend to support the filters we need?